src/app.rs

```diff
@@ -411,7 +411,7 @@ pub fn build_app() -> App<'static, 'static> {
         .arg(
             Arg::with_name("changed-within")
                 .long("changed-within")
-                .alias("change-newer-than")
```

Comment:
I think removing this alias isn't backwards compatible. Why not keep both aliases?